javascripts/discourse/controllers/gif.js

```diff
@@ -192,7 +195,9 @@ export default Controller.extend(ModalFunctionality, {
         );
         this.currentGifs.addObjects(images);
       } catch (error) {
-        bootbox.alert(error);
+        this.dialog.alert({
+          message: htmlSafe(error),
```

Comment:
Are the errors actually "html safe"? 😃

Reply:
You're right. I removed the HTML altogether.